gtest_parallel.py

```diff
@@ -177,11 +177,12 @@ class TaskManager(object):
   Logger, TestResults and TestTimes classes, and in case of failure, retries the
   test as specified by the --retry_failed flag.
   """
-  def __init__(self, times, logger, test_results, times_to_retry,
+  def __init__(self, times, logger, test_results, task_factory, times_to_retry,
```

Comment:
Is this a factory, or should this be called task_class? Python kinda makes this the same thing. WDYT?